src/components/EditAlbum.tsx

```diff
@@ -0,0 +1,74 @@
+"use client";
+
+import { addAlbum, editAlbum } from "@/actions";
+import { Album, genreEnum } from "@/db/schema";
+import { useRouter } from "next/navigation";
+import { useState } from "react";
+import toast from "react-hot-toast";
+
+export default function EditAlbum({ album }: { album: Album }) {
+  const [title, setTitle] = useState(album.title);
+  const [artist, setArtist] = useState(album.artist);
+  const [genre, setGenre] = useState(album.genre);
+  const [button, setButton] = useState(true);
```

Comment:
this variable is named badly, i have actually no idea what it does right now. Think of better naming, maybe something like:
```suggestion
  const [showEditButton, setShowEditButton] = useState(true);
```